python/scripts/daac_data_download_python/README.md

```diff
@@ -0,0 +1,58 @@
+# How to Access the LP DAAC From Data Pool and NASA Earthdata Cloud
+---
+# Objective:
+The `DAACDataDownload.py` script demonstrates how to configure a connection to download data directly in Python from an Earthdata Login-enabled server, specifically the [LP DAAC Data Pool](https://www.earthdata.nasa.gov/learn/use-data/tools) and [NASA Earthdata Cloud](https://www.earthdata.nasa.gov/). The script is a command line executable, where a user will submit either a single URL to a file to be downloaded, or the location of a csv or text file containing multiple URLs to be downloaded, and a desired directory to download files to. 
+
+
+The script begins with authenticating and then downloads the URL(s) that you provided. If multiple URLs are included in the file list, the script loops through URLs and downloads 8 files at a time using the `earthaccess` multithreading download method to speed up the download process. If the files provided in your list exist in the provided directory, the script skips those files. The output file name will be the same as the input file name.   
+
+---
+## Prerequisites/Setup Instructions  
+
+### Environment Setup 
+
+Instruction for setting up a compatible environment is available at: <https://github.com/nasa/LPDAAC-Data-Resources/blob/main/setup/setup_instructions_python.md>
```

Comment:
I think we need to write a separate set of instructions for this, since there are so many fewer python libraries required for this script.